sources/org.osbuild.ostree

```diff
@@ -91,30 +90,10 @@ class OSTreeSource(sources.SourceService):
     def fetch_one(self, checksum, desc):
         commit = checksum
         remote = desc["remote"]
-        url = remote["url"]
-        gpg = remote.get("gpgkeys", [])
-        uid = str(uuid.uuid4())
-
-        remote_add_args = []
-        if not gpg:
-            remote_add_args = ["--no-gpg-verify"]
-
-        if "contenturl" in remote:
-            remote_add_args.append(f"--contenturl={remote['contenturl']}")
-
-        if remote.get("secrets", {}).get("name") == "org.osbuild.rhsm.consumer":
-            secrets = Subscriptions.get_consumer_secrets()
-            remote_add_args.append(f"--set=tls-client-key-path={secrets['consumer_key']}")
-            remote_add_args.append(f"--set=tls-client-cert-path={secrets['consumer_cert']}")
-
-        ostree.cli("remote", "add",
-                   uid, url,
-                   *remote_add_args,
-                   repo=self.repo)
-
-        for key in gpg:
-            ostree("remote", "gpg-import", "--stdin", uid,
-                   repo=self.repo, _input=key)
+        # This is a temporary remote so we'll just use a random name
+        name = str(uuid.uuid4())
```

Comment:
Ah, I see, this is where the name/uid confusion comes from, it later changes name.

Reply:
I think the confusion is probably because I didn't split the commits appropriately. I fixed it now so that the name change happens in the next commit (where it should have been in the first place). Does that help?